src/handlers.js

```diff
@@ -116,16 +119,23 @@ module.exports = Handlers;
 
 /**
  * Post updates to GitHub, when the status of a task changes.
- * TaskCluster States: http://docs.taskcluster.net/queue/exchanges/
+ * TaskCluster States: https://docs.taskcluster.net/reference/platform/queue/references/events
  * GitHub Statuses: https://developer.github.com/v3/repos/statuses/
  **/
 async function statusHandler(message) {
   let taskGroupId = message.payload.taskGroupId || message.payload.status.taskGroupId;
-  debug(`handling state change for task-group ${taskGroupId}`);
+  debug(`Handling state change for task-group ${taskGroupId}`);
 
-  let build = await this.context.Builds.load({
-    taskGroupId,
-  });
+  try {
+    debug('Trying to load the build...');
```

Comment:
Has this section failed frequently?  If not, let's remove the try/catch and debugging.  An exception here will be handled by `callHandler`.